include/SDL3/SDL_gpu.h

```diff
@@ -1,4 +1,4 @@
-/*
+﻿/*
```

Comment:
BOM added again...

Reply:
I don't know why my editor keeps doing this, I'll figure it out.

Reply:
@thatcosmonaut, can you disable that in your editor?

Reply:
I think I figured it out, hopefully this isn't some silly thing that Git for Windows is doing.

Reply:
Perhaps it's time we add `charset: 'utf-8'` to our `.editorconfig`?
Lots of IDEs support it.

Reply:
Sure, sounds good